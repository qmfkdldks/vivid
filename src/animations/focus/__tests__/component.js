import React from "react";
import { shallow } from "enzyme";
import Focus from "../component";

describe("Focus", () => {
  it("should render successfully", () => {
    const component = shallow(<Focus>text</Focus>);
    expect(component).toMatchSnapshot();
  });
});
