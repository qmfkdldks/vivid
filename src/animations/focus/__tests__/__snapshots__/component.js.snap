// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Focus should render successfully 1`] = `
<Styled(Component)
  transition={
    Object {
      "duration": 1,
      "ease": "easeIn",
    }
  }
  variants={
    Object {
      "start": Object {
        "filter": Array [
          "blur(0px)",
          "blur(3px)",
          "blur(0px)",
          "blur(7px)",
          "blur(0px)",
          "blur(0px)",
        ],
      },
    }
  }
>
  text
</Styled(Component)>
`;
